src/cascadia/TerminalSettingsModel/defaults.json

```diff
@@ -382,6 +382,7 @@
         // Clipboard Integration
         { "command": { "action": "copy", "singleLine": false }, "keys": "ctrl+shift+c" },
         { "command": { "action": "copy", "singleLine": false }, "keys": "ctrl+insert" },
+        { "command": { "action": "copy", "singleLine": false }, "keys": "enter" },

```

Comment:
This is potentially a spicy change 🌶️ 

I know conhost did this in the past. I feel like we've discissed this in the past on the repl, but I can't find the discussion if we specifically said we'd add this to the defaults or not. 

We may also want to consider something like #13115 - what does that even mean? Is that something separate from a text selection?

Reply:
> We may also want to consider something like https://github.com/microsoft/terminal/issues/13115 - what does that even mean? Is that something separate from a text selection?

My vision for #13115 (which isn't even specced tbh), is that it's a way to navigate within mark mode. It might even be something as simple as <kbd>[shift]+tab</kbd> to move to the next/previous hyperlink. That 100% needs to be specc'd though.

> This is potentially a spicy change 🌶️

Yeah, I don't remember the discussion either :/. The paths forward I see are...
1. bind `copy()` to <kbd>Enter</kbd> in defaults.json (configurable)
2. make <kbd>Enter</kbd> copy the text when in mark mode (non-configurable)
3. don't even consider <kbd>Enter</kbd> entirely. If the user wants it, the user can bind it

Dustin seemed to be a little confused when using mark mode saying that there's no clear way to copy the selection. I think this stems from him using `copyOnSelect`, which seems fair because you never really had to think about copying your selection before. But we can't copy the selection _every time_ the selection changes in mark mode. That would totally mess up clipboard history for people who use it. :/

We could pull this part out and discuss it at sync on Monday? It's a one-liner with no risk. Really just "political" reasons to decide to add it in or not.

Reply:
Funny enough, now I'm thinking that we don't add this as a keybinding in defaults.json. Maybe we should make it non-configurable to allow for [this](https://github.com/microsoft/terminal/issues/6649#issuecomment-1128186887).

That way, <kbd>Enter</kbd> copies when in mark mode, but opens the URL when it's selected/found by tabbing through he buffer in mark mode. It would then go back to copying if the user modified the selection.

Reply:
Putting it in defaults.json allows the user to unbind it from both behaviors, but still gives us the freedom to change it (as opposed to inserting it in usersettings.json). We don't need to make the decision _now_, because defaults.json is entirely under our control across app updates.

Reply:
Have we considered adding a `mode` option to key bindings that would only apply when enabled? They could correspond to this enum https://github.com/microsoft/terminal/pull/13360#discussion_r905607764

Alacritty has the same concept (vi mode is mark mode) https://github.com/alacritty/alacritty/blob/master/alacritty/src/config/bindings.rs